t/plugin/ext-plugin/extra-info.t

```diff
@@ -20,6 +20,7 @@ repeat_each(1);
 no_long_string();
 no_root_location();
 no_shuffle();
+log_level("info");
```

Comment:
The default level is already info? https://github.com/apache/apisix/blob/a47d05a9a32db1ba7de6bda68c67cf0008f8442c/t/APISIX.pm#L24